.github/workflows/build.sh

```diff
@@ -3,32 +3,28 @@
 # Usage:
 # ./.github/workflows/build.sh 
 
-# this script will modified upper directory istio-proxy repo
-# if self-used should attention that whether the upper directory have its own developing istio.proxy
 #first build should give permission to docker volumes
 #sudo chmod -R 777 /var/lib/docker/volumes
 
 set -e
 
 UPDATE_BRANCH=${UPDATE_BRANCH:-"master"}
-cd ..
 rm -rf istio-proxy
 git clone -b ${UPDATE_BRANCH} https://github.com/intel/istio-proxy.git
-cp -rf istio/ istio-proxy/ 
-cd istio-proxy
 
+pushd istio-proxy
 # Update to release branch as intel/envoy has updated.
 ./scripts/update_envoy.sh
 BUILD_WITH_CONTAINER=1 make build_envoy 
 BUILD_WITH_CONTAINER=1 make exportcache
+popd
 
-cd istio
+# sgx build container would cause build proxyv2 image failed in release-1.15
```

Comment:
Can you explain this bit more -- how the SGX build container would cause the build to fail? Or how the SGX build will work without the container?

Reply:
Hmm, this makes no sense to me. The SGX build container was working just fine and now it does not and we just don't use it.

Reply:
Build binary needs SGX_BUILD_CONTAINER in istio-proxy but when build proxyv2 image in istio repo, SGX_BUILD_CONTAINER leads build failed.

Reply:
In internal version build, we only define IMG when build envoy binary.
But here we define IMG env in build.yml
https://github.com/intel/istio/blob/bd196add2c6a7fd9dc63a0172726f154f06d9aa4/.github/workflows/build.yml#L33
and if it is not unset it would affect build proxyv2 image in istio.